src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,695 @@
+    },
+    "groups": []
+  }
+  ```
+  </Collapser>
+  
+  <Collapser title="400 Bad Request">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "scimType": "invalidValue",
+    "detail": "Validation failed: Username can't be blank",
+    "status": "400"
+  }
+  ```
+  </Collapser>
+  
+  <Collapser title="409 Conflict">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "scimType": "uniqueness",
+    "detail": "Resource already exists",
+    "status": "409"
+  }
+  ```
+  </Collapser>
+</CollapserGroup>
+
+## Create groups in your authentication domain [#create-groups]
+
+You can use the SCIM API to send a `POST` request to `/scim/v2/Groups` to create a group. The only group attribute **required** is:
+* `displayName` The group name.
+
+```
+curl -X 'POST' -H 'Content-Type: application/json' -H "Authorization: Bearer $<var>YOUR_TOKEN</var>" 'https://scim-provisioning.service.newrelic.com/scim/v2/Groups' --data-binary @- <<EOF
+{
+    "schemas": ["urn:ietf:params:scim:schemas:core:2.0:Group"],
+    "displayName": "Example Group"
+}
+EOF
+```
+
+<Callout variant="important">
+Take note of the returned user `id`. To update a group or its members in the future you will need to supply the same ID with the request.
+</Callout>
+
+### Example responses
+
+<CollapserGroup>
+  <Collapser title="201 Created">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:schemas:core:2.0:Group"
+    ],
+    "id": "df2b9a04-0426-4a3e-bf5f-54d5341f4e5b",
+    "displayName": "Example Group",
+    "meta": {
+      "resourceType": "Group",
+      "created": "2021-07-20T22:41:50.414Z",
+      "lastModified": "2021-07-20T22:41:50.414Z"
+    },
+    "members": []
+  }
+  ```
+  </Collapser>
+  
+  <Collapser title="400 Bad Request">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "scimType": "invalidValue",
+    "detail": "Validation failed: Display name can't be blank",
+    "status": "400"
+  }
+  ```
+  </Collapser>
+  
+  <Collapser title="409 Conflict">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "scimType": "uniqueness",
+    "detail": "Resource already exists",
+    "status": "409"
+  }
+  ```
+  </Collapser>
+</CollapserGroup>
+
+## View users and groups in your authentication domain [#view]
+
+After you've created some users and groups you'll see them in the [User management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where). You can also retrieve them from the SCIM API.
+
+In this tutorial we'll search for specific users and groups, but that's not the only way to view users and groups. Refer to the [SCIM  API reference](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management/) and [RFC 7644](https://datatracker.ietf.org/doc/html/rfc7644) for all the available query options.
```

Comment:
This isn't accurate. It's not "we" who are doing the tutorial. It's the reader "you" who is doing the tutorial.